plugin-server/src/main/pluginsServer.ts

```diff
@@ -221,6 +225,29 @@ export async function startPluginsServer(
             await piscina!.broadcastTask({ task: 'sendPluginMetrics' })
         })
 
+        serverInstance.kafkaSuccessiveHealthchecksFailed = 0
+        if (hub!.kafka) {
+            kafkaHealthcheckJob = schedule.scheduleJob('* * * * *', async () => {
```

Comment:
Not sure if using a scheduled job running every minute is the way to go. I would suggest a background thread to constantly do the check

Reply:
It could, although in that case with one failure we'd mark it all as unhealthy, but the group might be rebalancing, or need a rejoin, etc.

Although as I write this I realize that the system running the healthcheck can deal with that via retry specs. I'll move it over to just do this when the endpoint is hit!

Reply:
one thing is that the timeout for the healthcheck endpoint might have to get pretty long to cope with this...